tests/unit/test_exceptions.py

```diff
@@ -14,9 +14,67 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-from cylc.flow.exceptions import CylcError
+from textwrap import dedent
+
+from cylc.flow.exceptions import (
+    CylcError,
+    HostSelectException,
+)
 
 
 def test_cylc_error_str():
     error = CylcError("abcd")
     assert str(error) == "abcd"
+
+
+def test_host_select_exception():
```

Comment:
IMO this ought to be one test per assert.

Reply:
I think it's ok to have multiple asserts in a test, the error comes back with a line number so I don't think there's really anything to gain by splitting them arbitrarily.

Reply:
Happy.